cgmanifest.json

```diff
@@ -140,6 +140,16 @@
                 }
             }
         },
+        {
+            "component": {
+                "type": "Maven",
+                "maven": {
+                    "artifactId": "lifecycle-process",
+                    "groupId": "androidx.lifecycle",
+                    "version": "2.2.0"
```

Comment:
We used one file from git, not the library from Maven

Reply:
Fixed